source/manual/auto-scaling-groups.html.md

```diff
@@ -28,41 +28,45 @@ the load.
 > ensure the number doesn't get put back to the old value if Terraform gets
 > deployed.
 
+1. Scaling up/down machines in AWS will trigger [Icinga alerts](icinga) so let
+   developers in`#govuk-2ndline` know you are about to do this.
+
 1. [Access the AWS Console][aws-console] and [go to the EC2 service][ec2-home].
 
-2. Select "Auto Scaling Groups" from the bottom of the menu on the left hand
+1. Select "Auto Scaling Groups" from the bottom of the menu on the left hand
```

Comment:
It looks like the numbering is using `1` for every step?

Reply:
Yes, using `1` for every step will make the numbers change automatically. So if we need to add an extra intermediate step in the future, we don't have to change them all but we can simply add it using a `1`.

Reply:
I had no idea! Thanks @AlanGabbianelli .